src/app/recipes/recipes.component.ts

```diff
@@ -18,17 +21,22 @@ export class RecipesComponent implements AfterViewInit {
 
   @ViewChild(RecipesListComponent, { static: true }) recipesListComponent!: RecipesListComponent;
 
-  recipes$ = this._recipesService.getAll();
+  recipes$ = this._store.select(selectRecipes);
 
   constructor(protected router: Router,
               private _recipesService: HttpRecipesService,
-              private _recipeDialogService: RecipeDialogService) {
+              private _recipeDialogService: RecipeDialogService,
+              private _store: Store) {
   }
 
   ngAfterViewInit(): void {
     this.recipesListComponent.selection$.pipe(
       switchMap(this.onSelectionChanged)
     ).subscribe();
+
+    this._recipesService.getAll().subscribe((recipes) =>
```

Comment:
When using store, you should avoid injecting services into components. You should dispatch action, that through effect (https://ngrx.io/guide/effects) will call http service. Best strategy for fetching data is to create two actions: GetRecipes and GetRecipesSuccess. GetRecipes will call http service through effect and when data is returned from that service, GetRecipesSuccess will be dispatched which will set correct values in store (same way as retrievedBookList which you used).